Crash when setting x-axis labels in the new stats UI

While testing the new stats UI, I faced the following crash:
```
2019-08-28 10:18:51.834 19921-19921/com.woocommerce.android E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.woocommerce.android, PID: 19921
    org.greenrobot.eventbus.EventBusException: Invoking subscriber failed
        at org.greenrobot.eventbus.EventBus.handleSubscriberException(EventBus.java:527)
        at org.greenrobot.eventbus.EventBus.invokeSubscriber(EventBus.java:509)
        at org.greenrobot.eventbus.EventBus.invokeSubscriber(EventBus.java:501)
        at org.greenrobot.eventbus.HandlerPoster.handleMessage(HandlerPoster.java:67)
        at android.os.Handler.dispatchMessage(Handler.java:107)
        at android.os.Looper.loop(Looper.java:214)
        at android.app.ActivityThread.main(ActivityThread.java:7343)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:933)
     Caused by: java.lang.IndexOutOfBoundsException: Collection doesn't contain element at index 7.
        at kotlin.collections.CollectionsKt___CollectionsKt$elementAt$1.invoke(_Collections.kt:77)
        at kotlin.collections.CollectionsKt___CollectionsKt$elementAt$1.invoke(Unknown Source:6)
        at kotlin.collections.CollectionsKt___CollectionsKt.elementAtOrElse(_Collections.kt:107)
        at kotlin.collections.CollectionsKt___CollectionsKt.elementAt(_Collections.kt:77)
        at com.woocommerce.android.ui.mystore.MyStoreStatsView$StartEndDateAxisFormatter.getFormattedValue(MyStoreStatsView.kt:517)
        at com.github.mikephil.charting.components.AxisBase.getFormattedLabel(AxisBase.java:472)
        at com.github.mikephil.charting.components.AxisBase.getLongestLabel(AxisBase.java:458)
        at com.github.mikephil.charting.renderer.XAxisRenderer.computeSize(XAxisRenderer.java:79)
        at com.github.mikephil.charting.renderer.XAxisRenderer.computeAxisValues(XAxisRenderer.java:74)
        at com.github.mikephil.charting.renderer.XAxisRenderer.computeAxis(XAxisRenderer.java:67)
        at com.github.mikephil.charting.charts.BarLineChartBase.notifyDataSetChanged(BarLineChartBase.java:332)
        at com.github.mikephil.charting.charts.Chart.setData(Chart.java:304)
        at com.woocommerce.android.ui.mystore.MyStoreStatsView.updateChartView(MyStoreStatsView.kt:362)
        at com.woocommerce.android.ui.mystore.MyStoreStatsView.updateView(MyStoreStatsView.kt:286)
        at com.woocommerce.android.ui.mystore.MyStoreFragment.showStats(MyStoreFragment.kt:208)
        at com.woocommerce.android.ui.mystore.MyStorePresenter.onWCRevenueStatsChanged(MyStorePresenter.kt:174)
```

--- a/WooCommerce/src/main/kotlin/com/woocommerce/android/ui/mystore/MyStoreStatsView.kt
+++ b/WooCommerce/src/main/kotlin/com/woocommerce/android/ui/mystore/MyStoreStatsView.kt
@@ -514,7 +514,12 @@ class MyStoreStatsView @JvmOverloads constructor(ctx: Context, attrs: AttributeS
             return when (value) {
                 axis.mEntries.first() -> getStartValue()
                 axis.mEntries.max() -> getEndValue()
-                else -> getLabelValue(chartRevenueStats.keys.elementAt(round(value).toInt() - 1))
+                else -> {
+                    val index = round(value).toInt() - 1
+                    return if (index > 0 && index < chartRevenueStats.keys.size - 1) {
+                        getLabelValue(chartRevenueStats.keys.elementAt(index))
+                    } else ""
+                }
             }
         }
 

